core/src/main/java/org/apache/servicecomb/core/exception/DefaultExceptionProcessor.java

```diff
@@ -0,0 +1,206 @@
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.servicecomb.core.exception;
+
+import static javax.ws.rs.core.Response.Status.BAD_REQUEST;
+import static javax.ws.rs.core.Response.Status.INTERNAL_SERVER_ERROR;
+import static javax.ws.rs.core.Response.Status.TOO_MANY_REQUESTS;
+import static org.apache.commons.lang3.exception.ExceptionUtils.getStackTrace;
+import static org.apache.servicecomb.core.exception.ExceptionCodes.GENERIC_SERVER;
+import static org.apache.servicecomb.swagger.invocation.InvocationType.CONSUMER;
+
+import java.util.Comparator;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import javax.ws.rs.core.Response.StatusType;
+
+import org.apache.servicecomb.config.inject.InjectProperties;
+import org.apache.servicecomb.config.inject.InjectProperty;
+import org.apache.servicecomb.core.Invocation;
+import org.apache.servicecomb.foundation.common.concurrent.ConcurrentHashMapEx;
+import org.apache.servicecomb.foundation.common.utils.ExceptionUtils;
+import org.apache.servicecomb.foundation.common.utils.SPIServiceUtils;
+import org.apache.servicecomb.swagger.invocation.Response;
+import org.apache.servicecomb.swagger.invocation.exception.CommonExceptionData;
+import org.apache.servicecomb.swagger.invocation.exception.ExceptionFactory;
+import org.apache.servicecomb.swagger.invocation.exception.InvocationException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+@InjectProperties(prefix = "servicecomb.invocation.exception")
+public class DefaultExceptionProcessor implements ExceptionProcessor {
+  private static final Logger LOGGER = LoggerFactory.getLogger(DefaultExceptionProcessor.class);
+
+  public static final int ORDER = Integer.MAX_VALUE;
+
+  @SuppressWarnings("unchecked")
+  private final List<ExceptionConverter<Throwable>> converters = SPIServiceUtils
+      .getOrLoadSortedService(ExceptionConverter.class).stream()
+      .map(converter -> (ExceptionConverter<Throwable>) converter)
+      .collect(Collectors.toList());
+
+  @InjectProperty(
+      prefix = "servicecomb",
+      keys = {
+          "invocation.exception.print-stack-trace",
+          "exception.invocation.print-stack-trace"

```

Comment:
why add two configs?

Reply:
servicecomb.exception.invocation.print-stack-trace is for compatible  

others planed same to 
```
sevicecomb.invocation.timeout.check.enabled
sevicecomb.invocation.timeout.check.strategy
sevicecomb.invocation.exception.print-stack-trace
sevicecomb.invocation.exception.print-rate-limit
```

Reply:
not released, no need to make compatible

Reply:
done